src/components/home/sections/BoardMembers.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import './volunteer.scss'
+import './boardmembers.scss'
+/*  IMAGES   */
+import johnson from '../../../images/johnson.jpg'
+import kim from '../../../images/kim.jpg'
+import timKim from '../../../images/tim-kim.jpg'
+import green from '../../../images/green.jpg'
+import roger from '../../../images/roger.jpg'
+export const BoardMembers = () => {
+  return (
+    <div className='main'>
+      <div className='volunteer-content'>
+
+        <h1 className='volunteer-title'>Board Members</h1>
```

Comment:
Is this class name being used in css anywhere?

Reply:
If you are referring to className 'main' then yes. The others were created before I joined the team.

Reply:
This one is about the "volunteer-title". Same as above.